util/src/main/java/com/google/cloud/hadoop/util/RetryHttpInitializer.java

```diff
@@ -81,6 +82,9 @@
   // Read timeout, in milliseconds.
   private final int readTimeoutMillis;
 
+  // HTTP request headers.
+  private final Map<String, String> headers;
```

Comment:
Change type to `ImmutableMap` to make sure that this map stays immutable after initialization.